src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,532 @@
+# -*- coding: utf-8 -*-
+
+"""This script downloads and parses IntAct data and maps the interaction types to BEL."""
+
+from typing import Dict, Iterable, List
+from zipfile import ZipFile
+
+import pandas as pd
+from protmapper.uniprot_client import get_mnemonic
+from tqdm import tqdm
+
+import pybel.dsl
+from bio2bel.utils import ensure_path
+from pybel import BELGraph
+
+
+#: Relationship types in IntAct that map to BEL relation 'increases'
+INTACT_INCREASES_ACTIONS = {
+    'phosphorylation reaction',
+    'sumoylation reaction',
+    'methylation reaction',
+    'transglutamination reaction',
+    'ubiquitination reaction',
+    'acetylation reaction',
+    'adp ribosylation reaction',
+    'neddylation reaction',
+    'hydroxylation reaction',
+    'phosphotransfer reaction',
+    'glycosylation reaction',
+    'palmitoylation reaction',
+    'oxidoreductase activity electron transfer reaction',
+    'amidation reaction',
+    'dna strand elongation',
+    'isomerase reaction',
+    'isomerization  reaction',
+    'proline isomerization  reaction',
+    'sulfurtransfer reaction',
+    'deamination reaction',
+    'ampylation reaction',
+    'aminoacylation reaction',
+    'myristoylation reaction',
+    'lipid addition',
+}
+
+#: Relationship types in IntAct that map to BEL relation 'decreases'
+INTACT_DECREASES_ACTIONS = {
+    # decreases
+    'deubiquitination reaction',
+    'protein cleavage',
+    'cleavage reaction',
+    'deacetylation reaction',
+    'lipoprotein cleavage reaction',
```

Comment:
would yo please add code comments explaining what lipoprotein cleavage reaction, dna cleavage, rna cleavage, and lipid cleavage mean? does BioGRID have a page that explains these? if it does, link them

Reply:
BioBRID uses the [Molecular Interactions Controlled Vocabulary](https://www.ebi.ac.uk/ols/ontologies/mi)

Reply:
I added the definitions for the interactions from the molecular interactions controlled vocabulary to the code, modified the target for the 'lipid cleavage' and 'lipoprotein cleavage reaction'

Reply:
Closed ee543c28b4e4f0c16aafe338ae138bdcf24b7908